package.json

```diff
@@ -132,6 +132,12 @@
 					"groups.json"
 				],
 				"url": "./syntaxes/imljson/schemas/groups.json"
+			},
+			{
+				"fileMatch": [
+					"makecomapp.json"
```

Comment:
wrong file name? makecomapp.types.ts

Reply:
It is correct. It defines the simulated file name. It means the testing file opened in editor will have this name.